### ISSUE TITLE
Add term metaphyseal undermodeling

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10442]
**Preferred term label:**
Metaphyseal undermodelling

**Synonyms**
Undermodelled metaphyses, Metaphyseal modeling defect

**Definition (free text, please give PubMed ID)**
Abnormal modeling of one or more metaphysis, which leads to changes in bone mass and shape.
PMID34484289

**Parent term (use hpo.jax.org/app)**
Abnormal metaphysis morphology HP:0000944

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bruck syndrome type 2 (OMIM609220) PMID15523624
Craniometadiaphyseal dysplasia (OMIM269300) PMID34484289

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
The definition does not actually say what the problem is -- can we describe this in more detail?

**Comment by azankl** [comment]
@pnrobinson how about:
lack of the normal diaphyseal constriction (thinning of long bones toward the midshaft area) giving the submetaphyseal areas an abnormally wide appearance

**Comment by pnrobinson** [comment]
@azankl 

Adding new term: 
- ID: HP:6001098
- label: Metaphyseal undermodelling
- parents: [Abnormal metaphysis morphology (HP:0000944)](https://hpo.jax.org/app/browse/term/HP:0000944)
- pmid: PMID:34484289
- def: Lack of the normal diaphyseal constriction (thinning of long bones toward the midshaft area) giving the submetaphyseal areas an abnormally wide appearance.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10442
- synonyms: Undermodelled metaphyses(NONE)|Metaphyseal modeling defect(NONE)
